whois-api/src/main/java/net/ripe/db/whois/api/autocomplete/AutoCompleteSearch.java

```diff
@@ -0,0 +1,14 @@
+package net.ripe.db.whois.api.autocomplete;
+
+import net.ripe.db.whois.common.rpsl.AttributeType;
+import net.ripe.db.whois.common.rpsl.ObjectType;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+public interface AutoCompleteSearch {
```

Comment:
Minor: autocomplete is one word so use ```AutocompleteSearch``` camel-casing. This is not easy to fix on OSX as the filename is case-insensitive